processor/metricstarttimeprocessor/config.go

```diff
@@ -4,15 +4,36 @@
 package metricstarttimeprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstarttimeprocessor"
 
 import (
+	"fmt"
+	"time"
+
 	"go.opentelemetry.io/collector/component"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstarttimeprocessor/internal/truereset"
 )
 
 // Config holds configuration of the metric start time processor.
-type Config struct{}
+type Config struct {
+	Strategy   string        `mapstructure:"strategy"`
+	GCInterval time.Duration `mapstructure:"gc_interval"`
```

Comment:
Shouldn't the `gc_interval` configuration option be documented?

Also it looks like this option is specific to the `true_reset_point` strategy and not a general processor option?

Reply:
`gc_interval` will be used by other strategies once they are added.  @ridwanmsharif can you document gc_interval in a follow-up?